src/include/configuration/configuration_manager.h

```diff
@@ -0,0 +1,133 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// configuration_manager.h
+//
+// Identification: src/include/configuration/configuration_manager.h
+//
+// Copyright (c) 2015-2017, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+#pragma once
+
+#include "type/types.h"
+#include "common/exception.h"
+#include "common/printable.h"
+
+namespace peloton {
+namespace configuration {
+  class Config;
+}}
+using peloton::configuration::Config;
+
+namespace peloton {
+namespace configuration {
+
+class Config {
```

Comment:
Shouldn't this be `ConfigurationUtil`. It's not obvious from just `Config` that it is a helper class.

Also, put it in a separate file. Don't combine it with the ConfigurationManager.